content/blog/2024/oci-manifest-capability.md

```diff
@@ -57,6 +58,24 @@ pub extern "C" fn wapc_init() {
     register_function("protocol_version", protocol_version_guest);
 }
 
+fn print_manifest(manifest : OciManifestResponse) {
```

Comment:
I would prefer the inline code for an example, it is more streamlined. If not, I would put `print_manifest()` after `validate()`.

Reply:
Latest example removes this code anyways. Marking as resolved